docs/source/conf.py

```diff
@@ -25,6 +25,7 @@
 # The full version, including alpha/beta/rc tags
 release = '0.1'
 
+numfig = True
```

Comment:
Curious, what is this being added for?

Reply:
This enables the use of _:numref:`%s <target>`_ syntax to refer to numbered items like tables, figures, and sections of the documentation. For some reason, we are still having trouble getting crosslinks to render using both :numref: and :ref:, and I have not had a chance to explore other reasons why that might be, but this line is definitely required for :numref: to work, in addition to whatever other changes may be required.

Reply:
Gotcha, thanks for explaining that @gspetro-NOAA !

Reply:
I have removed the noaacsp png's under _static dir, as we have stored them in Wiki and use links now.